eng/versioning/version_client.txt

```diff
@@ -445,7 +445,6 @@ unreleased_com.azure:azure-core;1.42.0-beta.2
 # beta_<groupId>:<artifactId>;dependency-version
 # note: Released beta versions will not be manipulated with the automatic PR creation code.
 beta_com.azure:azure-communication-common;1.3.0-beta.1
+beta_com.azure:azure-data-appconfiguration;1.5.0-beta.1
```

Comment:
https://repo1.maven.org/maven2/com/azure/azure-data-appconfiguration/1.5.0-beta.1/

Remove this line since beta.1 already released.

Reply:
@mssfang, we tried to do it without this and it doesn't work. We were running into so build issue.